delivery_schenker/models/delivery_carrier.py

```diff
@@ -429,15 +432,15 @@ def _schenker_shipping_information_without_packages(self, picking):
                 "dgr": False,
                 "cargoDesc": picking.name,
                 # For a more complex solution use packaging properly
-                "grossWeight": self._schenker_shipping_information_round_weight(
-                    weight / picking.number_of_packages
-                ),
+                "grossWeight": self._schenker_shipping_information_round_weight(weight),
                 "volume": self._schenker_shipping_information_round_volume(
-                    self._schenker_shipping_information_without_packages_volume(picking)
+                    self._schenker_shipping_information_without_packages_volume(
+                        move_lines_without_package
+                    )
                 ),
                 "packageType": self.schenker_default_packaging_id.shipper_package_code,
                 "stackable": self.schenker_default_packaging_id.schenker_stackable,
-                "pieces": picking.number_of_packages,
+                "pieces": 1,
```

Comment:
do you really need to remove the dependency for this module's tests to work?

Reply:
Yes because without the module `base_delivery_carrier_label` installed before there is no package, so `number_of_packages` is 0. 
Also this part of code is not correct, because in a scenario where some moves are packed and one isn't, this will send the wrong amount for pieces. Here in this method always `1` needs to be the value for pieces.